packages/salesforcedx-visualforce-language-server/tsconfig.json

```diff
@@ -1,7 +1,6 @@
 {
   "extends": "../../tsconfig.common.json",
   "compilerOptions": {
-    "moduleResolution": "node",
```

Comment:
Not sure on this one...It's a stretch but any memory of changing this @RitamAgrawal ?